src/backend/api/Fusion.Resources.Api/Controllers/Requests/Requests/CreateRequestComment.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentValidation;
+using System.Text.Json.Serialization;
+
+namespace Fusion.Resources.Api.Controllers
+{
+    public class CreateRequestComment
+    {
+        public string Content { get; set; } = null!;
+
+        [JsonConverter(typeof(JsonStringEnumConverter))]
+        public CommentOrigin Origin { get; set; }
```

Comment:
Should this not be resolved by the api? If User.AccountType == employee -> Company, else contractor?

Reply:
Of course! 🤦‍♂